tests/python/test_ndarray.py

```diff
@@ -567,24 +445,28 @@ def func(a: ti.any_arr(element_dim=1)):
     v = np.zeros((6, 10), dtype=np.int32)
     func(v)
     assert impl.get_runtime().get_num_compiled_functions() == 1
-    import torch
```

Comment:
Splitting this test violates its original intention. I suggest using a numpy array instead here if torch is not installed.